packages/typography/src/constants.tsx

```diff
@@ -0,0 +1,26 @@
+export const CaptionType = {
+  CAPTION1: 'caption1',
+} as const
+
+export const Paragraph = 'p'
+
+export const ParagraphType = {
+  P1: 'p1',
+  P2: 'p2',
+  P3: 'p3',
+  P4: 'p4',
+} as const
+
+export const DisplayType = {
+  DISPLAY1: 'display1',
+  DISPLAY2: 'display2',
+} as const
+
+export const HeadingType = {
```

Comment:
Can we either use lower camelCase or uppercase for constants? upper-cammelCase should be reserved for component names or classes.

On the FE we use uppercase, it might be better to do the same here. e.g.

```
HEADING.H1
```